docs/pages/machine-id/getting-started.mdx

```diff
@@ -55,9 +62,9 @@ binaries, including `teleport`, `tctl`, `tsh`, and `tbot`:
 - `tsh` is the client tool you will use to log in to the Teleport Cluster (steps 2/4 and 4/4)
 - `tbot` is the Machine ID tool you will use to associate a bot user with a machine (step 3/4)
 
-Machine ID is available starting from the Teleport `9.0.0` release. Download
-the appropriate Teleport package for your platform from our
-[downloads page](https://goteleport.com/teleport/download).
+Install Teleport on the host where you will run Machine ID:
```

Comment:
Why? You don't need teleport, only tbot.

Reply:
We can title it as `tbot`. This is the method to get `tbot`.

Reply:
I think we'd want to encourage them to use that method so they can consistently install and upgrade.

Reply:
```suggestion
Install Teleport, which includes `tbot`, on the host where you will run Machine ID:
```
Unless there's a way to install `tbot` on its own?